codecov.yml

```diff
@@ -3,3 +3,5 @@ coverage:
     project:
       default:
         informational: true
+    patch:
+      informational: true
```

Comment:
We somehow don't have a unit test file for the quite large `evaluation_executor.py`.  https://github.com/eth-easl/modyn/pull/490

@robinholzi Is there a reason? If it is no intentional please add one!

I will for the moment also make `patch` action informational. As it does not make sense for me to create a huge unit test only to cover these trivial changes.